src/connection.js

```diff
@@ -88,6 +88,24 @@ const GetInflationResult = struct({
   terminal: 'number',
 });
 
+/**
+ * EpochSchedule parameters
+ *
+ * @typedef {Object} EpochSchedule TODO - link to book terminology?
```

Comment:
No new TODOs please, fill in what you can from the main repo and otherwise leave the description blank